### ISSUE TITLE
Add tools to resources page?

### ISSUE THREAD

**Issue by clhenrick** [MaptimeNYC/maptimenyc.github.io#8]
Right now the resources page links to the Maptime HQ page. I think it would be useful to have some links to resources and brief descriptions. Eg: [Projection Wizard](http://projectionwizard.org/), CartoDB, QGIS, Leaflet, etc.

**Comment by bfreeds** [comment]
@clhenrick  I agree it would be useful to have the tools, if we are able to keep the list and descriptions updated.  We could organize it by tool type (Web Map Libraries, APIs, Geoprocessing Tools, Desktop GIS Tools...)

**Comment by clhenrick** [comment]
Cool, let's start a Google Doc with a list? Or we may be able to store it as a CSV or YML file then have Jekyll read it into the resources page.
